src/instruments/view/instrumentpaneltreemodel.cpp

```diff
@@ -567,8 +567,9 @@ AbstractInstrumentPanelTreeItem* InstrumentPanelTreeModel::buildPartItem(const P
 
 void InstrumentPanelTreeModel::updatePartItem(PartTreeItem* item, const Part* part)
 {
+    std::cout << "InstrumentPanelTreeModel::updatePartItem: title = \"" << part->partName().toStdString() << '"' << std::endl;
```

Comment:
better use `LOGD` or `qDebug()`

Reply:
I think it's better to just remove this log, because other developers don't need to see this information often

Reply:
Better not use at all! Was a left-over of some testing, removed.